### ISSUE TITLE
UI accessibility issue, app improvements and more

### ISSUE THREAD

**Issue by azurejoga** [Divested-Mobile/hypatia#97]
Hi everyone, I'm an independent security researcher and visually impaired.
I'm here opening this issue to report some accessibility bugs, application failures, improvements that we can implement, integrations and even an invitation!
* Used Android, Android 14 Samsung M15.
## accessibility improvements and UI issues and other bugs
* An accessibility issue that causes the Android screen reader, Google talkback, to not read the home screen correctly.
The main screen of the application where it shows carrying out operations such as statuses such as starting the scam, among others is being read in just one line which means that I have to listen to it all at once to understand what is happening and this is extremely  bad. To correct this I was going to suggest that the screen be restructured, just like the credits screen where everything is written on separate lines, that is, on a new line. Look at the credits screen and you'll see what I mean.  
In short, the Main screen, which is the most important screen, shows the text display in just one line, although visually it doesn't seem like that, for the screen reader it certainly is different.
* Problem deleting an item
When detecting malware in the notification bar when clicking delete, nothing is shown, at least nothing to my screen reader and honestly I get confused whether it was actually deleted or not.
* Sometimes the application stops working and a message appears on the screen, such as the application has stopped responding or the application has continuous crashes, in addition this appears both in the notification bar and as a popup. I have no idea how this happens, I'm just sure it happens if you leave the real-time Scam enabled, if you leave it disabled and do a Scam by clicking the button in the main UI it works fine, so I assume it's the method they are using for that.
## app improvements
1. Add a button to view history and when clicked, it opens a window, with the texts well structured on a new line showing everything that has already been done, in two tabs. The first is analyses, where it shows the date and time of the analyses/scams and when clicked it shows whether something was detected, how many files were scanned, etc... with the possibility of deleting, consulting or ignoring as requested by the user.  second tab, where it shows everything that has already been done in the application, such as a log record, example: 2024/10/29 Downloaded Server database XXXX
2. Add the possibility of choosing whether to receive notifications in the notification bar or a button to view the history in the notification bar or in the main UI of the application which, when clicked, takes you to the history As written above with the intended functionalities.
3. Allow the user to choose whether they want to receive malware notifications in the notification bar or if they just want to receive a notification with the view history button.
4. Allow user to disable notifications from notification bar
5. Allow user to add DNS lists, IPS lists, domain list to block them, like a firewall
6. Add other types of viruses, such as Ransomwares, Worms, backdoors, etc... and their hashes in the database
## An Invitation
I'm an independent developer / cyber security researcher and I would be very happy to help the opensource community, and I believe that many opensource applications are just not that famous because many people don't know *use* github, they don't know that open source applications they exist. and to that end, I'm here to invite developers, along with the community, to consider publishing the app on Google Playstore, where it would be done by me, in my developer account, the credits would all be given to the developer, I don't want anything for myself! the only purpose I want for this is to promote and help grow the app. And of course, it would be done at no cost! my only purpose will be to promote this beautiful app!   What do you think of that?
## Thank you!
Thanks for the app, and thanks for taking security to another level!
@SkewedZeppelin 
@chaoscalm 
@Tomoms 
@ovginkel 
@huuhaa

**Comment by azurejoga** [comment]
Any results on this?
